helm/harbor-operator-app/Chart.yaml

```diff
@@ -1,11 +1,20 @@
-apiVersion: v1
-appVersion: 0.5.1
-description: A Helm chart to deploy harbor-operator
-engine: gotpl
+apiVersion: v2
```

Comment:
Does upgrade from `v1` to this work? Maybe not important since we are not managing this app yet.